test/unit/dynamic-bootstrap-specs.js

```diff
@@ -44,7 +44,7 @@ describe('dynamic bootstrap', function () {
     rimraf(process.env.APPIUM_BOOTSTRAP_DIR)
 
       // first call: should create new bootstrap file
-      .then(function () { return prepareBootstrap(); })
+      .then(async function () { return await prepareBootstrap(); })
```

Comment:
Yes. Everything not in what is currently `resolve-deps.js` are worked on in my PR.